tagstudio/src/qt/modals/file_extension.py

```diff
@@ -61,11 +63,17 @@ def __init__(self, library: "Library"):
         self.mode_layout.setContentsMargins(0, 0, 0, 0)
         self.mode_layout.setSpacing(12)
         self.mode_label = QLabel()
-        self.mode_label.setText("List Mode:")
+        Translations.translate_qobject(self.mode_label, "ignore_list.mode.label")
         self.mode_combobox = QComboBox()
         self.mode_combobox.setEditable(False)
-        self.mode_combobox.addItem("Include")
-        self.mode_combobox.addItem("Exclude")
+        self.mode_combobox.addItem("")
+        self.mode_combobox.addItem("")
+        Translations.translate_with_setter(
+            lambda text: self.mode_combobox.setItemText(0, text), "ignore_list.mode.include"
+        )
+        Translations.translate_with_setter(
+            lambda text: self.mode_combobox.setItemText(0, text), "ignore_list.mode.exclude"
```

Comment:
Fix for setting the first item twice:
```suggestion
            lambda text: self.mode_combobox.setItemText(1, text), "ignore_list.mode.exclude"
```
![image](https://github.com/user-attachments/assets/ca690edd-37ef-460a-820c-8b64cbc22411)